server/arweave/router.ts

```diff
@@ -81,9 +59,17 @@ router.post('/upload',
 
     // shortcut for existing file without checking tx
     if (existingArweaveId) {
+      let numAssetsIds = [];
+      if (req.body.num === '1') {
```

Comment:
Would `req.body.num && req.body.num !== '0'` be more robust?

Reply:
Updated